### PR TITLE
Add worker dyno

### PR DESCRIPTION
**Pull request by unasuke**
sidekiq!

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+web: bin/rails server -p 3000
+worker: bundle exec sidekiq --queue default
```
Comment (okuramasafumi) [inline Procfile:2]: What if we want to use queue other than `default`?

Comment (unasuke) [reply Procfile:2]: Update this line

Comment (okuramasafumi) [reply Procfile:2]: OK.

**Review by okuramasafumi — APPROVED** [review]
🐾

**Pull request merged**